store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -362,12 +362,12 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
           FieldNameUpdatedHandler(pgu, truthCopyInfo, secondaryColInfo)
           (rebuildIndex, false,  truthCopyInfo, None)
```

Comment:
This should also probably be `refreshRollup`.  It won't happen in changesets generated via the public API, but it's totally possible to tell data coordinator "here are a bunch of rows, oh and change that fieldname too" in a single mutation script.